packages/neos-ui/src/Containers/LeftSideBar/NodeTree/Node/index.js

```diff
@@ -160,6 +161,11 @@ export default class Node extends PureComponent {
         };
     }
 
+    decodeLabel = compose(
```

Comment:
I think we can merge it as is, just a stylistic comment, since this `decodeLabel` is just an utility method and doesn't need access to `this`, it could be just written above the `class`, as a simple variable.

Reply:
@dimaip Okay, fair point. Will do that :)